database/migrations/20240212042820_forms.up.sql

```diff
@@ -0,0 +1,65 @@
+BEGIN;
+
+CREATE SCHEMA forms;
+
+CREATE TYPE forms.status_type AS ENUM ('DRAFT', 'PUBLISHED');
+
+CREATE TABLE forms.form (
+    id bigserial PRIMARY KEY,
+    content JSONB NOT NULL,
+    created_by_id INT NOT NULL,
+    owner_id INT NOT NULL,
+    "status" forms.status_type DEFAULT 'DRAFT',
+    created_at timestamp DEFAULT (NOW() AT TIME ZONE 'UTC'),
+    updated_at timestamp DEFAULT NULL
+);
+
+CREATE TABLE forms.metadata (
+    id bigserial PRIMARY KEY,
+    form_id INT NOT NULL,
+    is_deleted BOOLEAN NOT NULL DEFAULT FALSE,
+    accepting_responses BOOLEAN NOT NULL DEFAULT FALSE,
+    allow_guest_responses BOOLEAN NOT NULL DEFAULT TRUE,
+    allow_multiple_responses BOOLEAN NOT NULL DEFAULT FALSE,
+    send_confirmation_email_to_responde BOOLEAN NOT NULL DEFAULT FALSE,
+    send_submission_email_to_owner BOOLEAN NOT NULL DEFAULT FALSE,
+    valid_till timestamp,
+    created_at timestamp DEFAULT (NOW() AT TIME ZONE 'UTC'),
+    updated_at timestamp DEFAULT NULL
+);
+
+CREATE TABLE forms.responses (
+    id bigserial PRIMARY KEY,
+    response_by_id INT NOT NULL,
+    content JSONB NOT NULL,
+    form_id INT NOT NULL,
+    created_at timestamp DEFAULT (NOW() AT TIME ZONE 'UTC'),
+    updated_at timestamp DEFAULT NULL
```

Comment:
are we allowing edit response..? if yes UpdatedBy needed if not need to remove updated_at

Reply:
- we currently do not allow a response to be edited, but updatedBy will not be required as updating a response will be restricted to a single userID